### PR TITLE
152 add pre commit check for translations

### LINKED ISSUE #152: Add pre-commit check for translations
Run babel update (and possibly formatting), strip irrelevant info, in pre-commit hook and fail on diff
https://github.com/MinBZK/amt/blob/main/script/translate

It would be nice to also check if the PO (base) files do not contain the comment: #, fuzzy
Translations don't work if that comment exists, so it needs to be fixed prior to committing!

### PR DESCRIPTION
**Pull request by ChristopherSpelt**
# Description

Script to check if the translate update and compile are executed. Intended use is a pre-commit hook; it will check if staged .po files are up to date, if not it will show a message "Translations are not up to date." and the pre commit hook will fail. Note that the script runs the translate update and compile commands, so the script is idempotent. Running the pre commit hook twice on staged .po files will make the pre-commit hook pass.

Out of scope:
- check if all translation strings are present.

Resolves #152 

## Checklist

Please check all the boxes that apply to this pull request using "x":

-   [x] I have tested the changes locally and verified that they work as expected.
-   [x] I have followed the project's coding conventions and style guidelines.
-   [x] I have rebased my branch onto the latest commit of the main branch.
-   [x] I have squashed or reorganized my commits into logical units.
-   [x] I have read, understood and agree to the [Developer Certificate of Origin](../blob/main/DCO.md), which this project utilizes.

### CODE REVIEW

File: amt/locale/nl_NL/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16,7 +16,7 @@ msgstr ""
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=utf-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.16.0\n"
```
Comment (anneschuth) [inline amt/locale/nl_NL/LC_MESSAGES/messages.po:19]: How come this version is going down?

File: amt/locale/en_US/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -284,3 +284,7 @@ msgstr ""
 
 #~ msgid "Lastupdated "
 #~ msgstr ""
+
+#~ msgid " TEST "
+#~ msgstr ""
+
```
Comment (anneschuth) [inline amt/locale/en_US/LC_MESSAGES/messages.po:290]: Is this intended to be commited?

**Review by anneschuth — APPROVED** [review]
lgtm!

File: amt/locale/nl_NL/LC_MESSAGES/messages.po (hunk 1 above)
Comment (ChristopherSpelt) [reply amt/locale/nl_NL/LC_MESSAGES/messages.po:19]: Hmmm that's peculiar indeed. I am not sure why.

File: amt/locale/en_US/LC_MESSAGES/messages.po (hunk 2 above)
Comment (ChristopherSpelt) [reply amt/locale/en_US/LC_MESSAGES/messages.po]: No I removed it

**Review by anneschuth — APPROVED** [review]
lgtm

**Pull request merged**